modules/RTC/RTC.js

```diff
@@ -19,6 +19,12 @@ import VideoType from '../../service/RTC/VideoType';
 
 const logger = getLogger(__filename);
 
+/**
+ * The counter used to generated id numbers assigned to peer connections
+ * @type {number}
+ */
+let peerConnectionIdCounter = 1;
```

Comment:
Call me paranoid, but in contexts where the app is not reloaded, this could overflow. Maybe we should reset this to 1 once it reaches `Number.MAX_SAFE_INTEGER`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

Reply:
I don't think our app is able to stay that long without crashing ;) but yeah I can add that to both counters (there's another one for track IDs).